### ISSUE TITLE
Update to support Homey Pro 2023?

### ISSUE THREAD

**Issue by hjelmdal** [hilvarenbeek/nl.marcovandenhout.marantz#39]
Hi, 

any chance you would update this app to support Homey Pro 2023? 

Thanks a million!

**Comment by happyshow** [comment]
+1 vote for this.

**Comment by ghost** [comment]
+1 from me as well

**Comment by Not2Fast** [comment]
Just for your information: most Marantz receivers should work with the Denon AVR app which is available for Honey Pro 2023.

**Comment by hilvarenbeek** [comment]
Hi, so far no new maintainer has picked this up AFAIK and I do not own a Homey Pro. It is possible that you can get it to work via a HEOS app. My app uses a simple network connection to talk directly to the Marantz (or Denon) but the disadvantage of that is that there are model-specific commands. Besides me not having a Homey Pro and no recent D/M AVR, I’m currently suffering from medical issues which makes it even harder to find time for this. I’m sorry but don’t expect much from me in this repository. Thanks for showing interest though!

**Comment by happyshow** [comment]
Thanks for your work.

**Comment by hjelmdal** [comment]
> 

Sorry to hear! Hope you get well soon! Thanks for the reply 🙏

**Comment by ghost** [comment]
@hilvarenbeek, thanks for your efforts and hope you will get better soon.  

For me, the Denon AVR app seems to work fine, having an Marantz NR1603

**Comment by OttoWerse** [comment]
> Just for your information: most Marantz receivers should work with the Denon AVR app which is available for Honey Pro 2023.

Thank you so much for that info! The Denon App works great with my SR6015s, even better than the Marantz app ever did since it has full support for the 8K Input. The Marantz App can switch to it as "AUX 2", but would not know when the receiver was switched to it via remote etc., the Denon app does just that. I don't think maintaining two different Apps for Denon and Marantz even really makes sense since they are basically identical in regard of their network control protocol and HEOS functionality. They should just be merged into one HEOS or Sound United app so it is easier for new users to find it, no matter which brand plate they happened to put on the specific model they have 😉

**Comment by hilvarenbeek** [comment]
I agree and think Athom should remove my app from the list and point at the Denon app in stead. It could be renamed Denon/Marantz or something and development could continue there. I think this has been suggested before though, maybe someone on here is still in touch with Athom?

**Comment by renoopava** [comment]
+1 vote for this.

**Comment by fmvdb** [comment]
> Hi, so far no new maintainer has picked this up AFAIK and I do not own a Homey Pro. It is possible that you can get it to work via a HEOS app. My app uses a simple network connection to talk directly to the Marantz (or Denon) but the disadvantage of that is that there are model-specific commands. Besides me not having a Homey Pro and no recent D/M AVR, I’m currently suffering from medical issues which makes it even harder to find time for this. I’m sorry but don’t expect much from me in this repository. Thanks for showing interest though!

Thanks for replying and your effort put into the app. Hopefully you’ll recover.

And I can confirm the Denon AVR app works on my Marantz SR-6013

**Comment by hilvarenbeek** [comment]
Consolidating update/transfer efforts to issue #37
